### PR TITLE
Add date when the fields will be removed

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: opentelemetry/proto/metrics/v1/metrics.proto [hunk 1]
```diff
@@ -130,8 +130,10 @@ message Metric {
     // and `sum` fields.
     // 3. New receivers, which are aware of this change MUST convert these into
     // `gauge` and `sum` by using the provided as_int field in the oneof values.
+    // This field will be removed in ~3 months, on July 1, 2021.
```
Comment (jsuereth) [inline opentelemetry/proto/metrics/v1/metrics.proto:133]: Is this enough time for SDKs THEN collector?

Comment (bogdandrutu) [reply opentelemetry/proto/metrics/v1/metrics.proto:133]: Do you think 6 months will be better?

Comment (jmacd) [reply opentelemetry/proto/metrics/v1/metrics.proto:133]: I imagine the whole project is better off if we move quickly. I support 3 months.

**Review by jmacd — APPROVED** [review]

**Review by jsuereth — APPROVED** [review]

File: opentelemetry/proto/metrics/v1/metrics.proto (hunk 1 above)
Comment (jsuereth) [reply opentelemetry/proto/metrics/v1/metrics.proto:133]: We discussed this in the SiG.  I'll open an OTEP around attempting to use real-usage to determine migration timelines vs. hard dates and we can discuss my concerns on that.   This is a non-blocking concern on this PR.

**Pull request merged**